### DISCUSSION TITLE
What is "Catalog Sources" used for?

### DISCUSSION THREAD

**Discussion by AlbatorLaho** [GetOpenStore/OpenStore#28] [General]
I was wondering what "Catalog Sources" is supposed to be used for.
I would think (in the case of brew) it would be taps... but when I open it, none of my tapped repos are listed. Is that because I have to add them in Overt explicitly? or is this supposed to be used for something else?

**Comment by ThatsJustCheesy** [comment]
Hi @Albator11, you're right that it is meant to list all installed taps. That list should be auto-updated at app launch, but as you've already identified in [an issue (thanks!)](https://github.com/GetOvert/Overt/issues/30), it's failing to do so.

So your confusion is a direct result of this bug :) which I'll try to fix this weekend.
